ext/AgentsVisualizations/src/spaces/abstract.jl

```diff
@@ -183,6 +183,6 @@ function Agents.agent2string(agent::A) where {A<:AbstractAgent}
     return agentstring
 end
 
-Agents.convert_element_pos(::S, pos) where {S<:Agents.AbstractSpace} = Tuple(pos)
+Agents.convert_element_pos(s::S, pos) where {S<:Agents.AbstractSpace} = Tuple(pos[1:length(s.dims)])
```

Comment:
Hm, I think this is not long-term stable. We shouldn't access the internals of the space. It may not be guaranteed that a space has a field `.dims`. can we use the size space function instead?

https://github.com/JuliaDynamics/Agents.jl/blob/main/src/spaces/utilities.jl#L10